src/reducers/versions.tsx

```diff
@@ -458,13 +489,18 @@ export const selectCurrentVersionInfo = (
   return getVersionInfo(versions, versions.currentVersionId);
 };
 
-export const getMostRelevantEntryStatus = (
-  version: Version,
-  path: string,
-): VersionEntryStatus | void => {
+export const getMostRelevantEntryStatus = ({
+  version,
+  entryStatusMap,
+  path,
+}: {
+  version: Version;
+  entryStatusMap: EntryStatusMap;
+  path: string;
+}): VersionEntryStatus | undefined | null => {
```

Comment:
This no longer returns `null`, right? I think it did only in a previous version.